TESTING.asciidoc

```diff
@@ -628,7 +628,7 @@ If you want to run Elasticsearch and be able to remotely attach the process
 for debugging purposes from your IDE, can start Elasticsearch using `ES_JAVA_OPTS`:
 
 ---------------------------------------------------------------------------
-ES_JAVA_OPTS="-Xdebug -Xrunjdwp:server=y,transport=dt_socket,address=4000,suspend=y" ./bin/elasticsearch
+ES_JAVA_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005" ./bin/elasticsearch
```

Comment:
I think the `server=n` and `suspend=y` need to stay? That is what we have with the `--debug-jvm` build flag. @mark-vieira can you confirm?

Reply:
I'd be more apt to remove this section to be honest since above we describe using `run --debug-jvm`. If folks want to debug a Java application the only thing we should have to instruct them to is _where_ to add the JVM arguments. They can then configure it with or without suspend, on whatever port they wish. If you have to be hand held through this process then you're probably not going to be using a debugger.

So for me either a) remove this section completely or b) remove the example arguments and simply tell folks to add to `ES_JAVA_OPTS` (or link to [this documentation](https://www.elastic.co/guide/en/elasticsearch/reference/master/jvm-options.html)).

Reply:
I'm in favor of just removing this.

@qabbasi Are you running this for development purposes (ie with the Elasticsearch build)? Are you interested in changing this PR to remove this section as @mark-vieira suggested?